fix import for py3.10 compat: collections.Iterable -> collections.abc…

plicplocplicploc opened the pull request:
The `collections.Iterable` is broken in python 3.10, `collections.abc.Iterable` has to be used.

Commit:
fix import for py3.10 compat: collections.Iterable -> collections.abc.Iterable

tomol012 reviewed (changes requested):
base.py line 129 needs that changed as well. elif isinstance(raw_result, collections.Iterable): becomes elif isinstance(raw_result, collections.abc.Iterable):
After this change, it works well

esregnet0409 commented:
I would like to submit a suggestion. 
How about to check the backward compatibility issue? 
The try-exception expression is more suitable. 

try:
    collections = collections.abc
except AttributeError:
    pass

Commit:
Update base.py

I'm not sure why I had modified the `import` statement and not line 129. Reverted the `import` to a simple `import collections`, and instead changed the call from `collections.Iterable` to `collections.abc.Iterable` at line 129).

boazcstrike reviewed (approved):
We need this for our project too.
